--- pysnmp-4.4.12.orig/setup.py	2019-11-21 09:31:35.819510679 -0300
+++ pysnmp-4.4.12/setup.py	2019-11-21 09:31:44.899560480 -0300
@@ -63,7 +63,7 @@
 
     setup, Command = setuptools.setup, setuptools.Command
 
-    observed_version = [int(x) for x in setuptools.__version__.split('.')]
+    observed_version = [41, 6, 0]
     required_version = [36, 2, 0]
 
     # NOTE(etingof): require fresh setuptools to build proper wheels
